app/controllers/api/v1/events_controller.rb

```diff
@@ -1,7 +1,7 @@
 class Api::V1::EventsController < Api::V1::BaseController
   def show
-    event = Facebook::Event.new(params[:id])
-    event.load!
-    render json: event, status: 200
+    event = Facebook::Event.new(id: params[:id])
+    repository = Facebook::EventRepository.new
+    render json: repository.load!(event), status: 200
```

Comment:
Move the initialization of event object to a private method sounds good I didn't because the controller have only one action what make it pretty small.

You mean a `load` class like this one https://github.com/WarmApp/warmapp-api/pull/2/files#diff-58207611003444553aa71dcbd2ce92e9R2 ?